src/v2/Utils/getENV.tsx

```diff
@@ -1,12 +1,10 @@
-import { getAsyncLocalStorage } from "lib/asyncLocalWrapper"
 // eslint-disable-next-line no-restricted-imports
 import { GlobalData, data as sd } from "sharify"
 
 export function getENV(ENV_VAR: keyof GlobalData) {
   let envVar
   if (typeof window === "undefined") {
-    const asyncLocalStorage = getAsyncLocalStorage()
-    envVar = asyncLocalStorage.getStore()?.get(ENV_VAR) || process.env[ENV_VAR]
+    envVar = process.env[ENV_VAR]
```

Comment:
This doesn't...100% feel right but I'm sort of confusing myself.

If someone is using `getENV` (which is kind of the recommended approach up until now to access any request-specific thing 'safely') on the server for a request-specific local, `process.env` wouldn't be right, whereas if it was a local that was bootstrapped into the local storage (via middleware), it _would_ be correct.

It might not be an issue in practice, but overall this is why a holistic overview/refactor/simplification of accessing locals is needed!

Reply:
I'm actually fixing this now. Thank you cypress!